contributions/catalog/webapps/templates/contribute/home.html

```diff
@@ -215,6 +226,9 @@ <h2 id="title"> Contributions Catalog/Packager </h2>
             <button class="btn" id="talent_btn" onclick="get_url('talent')"> Talents</button>
         {% endif %}
 
+        <div class="footer">
+            <p>Catalog version {{ gittag }} &#124; &copy; University of Illinois Board of Trustees &#124; <a href="https://www.vpaa.uillinois.edu/resources/web_privacy">Privacy Policy</a> </p>
```

Comment:
```suggestion
            <p>Version {{ gittag }} &emsp; &#124; &emsp; &copy; University of Illinois Board of Trustees &emsp; &#124; &emsp; <a href="https://www.vpaa.uillinois.edu/resources/web_privacy">Privacy Policy</a> </p>
```
Suggesting some UI changes.